services/server/config/config.go

```diff
@@ -202,6 +210,28 @@ func (bc *BoltConfig) Validate() error {
 	}
 }
 
+const (
+	// ExporterTypeStdout represents stdout exporter
+	ExporterTypeStdout = "stdout"
+	// ExporterTypeOtlp represents the open telemetry exporter OTLP
+	ExporterTypeOtlp = "otlp"
+)
+
+// Validate openTelemetry config
+func (cfg *OpenTelemetryConfig) Validate() error {
+	switch cfg.ExporterName {
+	case ExporterTypeStdout:
+		return nil
```

Comment:
Do we really need to support stdout? containerd's stdout is currently used for logging as like all systemd-native daemons.

Reply:
It doesn't seem like we need to expose an exporter name at this point.
I don't see why people would need something other than OTLP, but if they do we can address that later if it comes up (e.g. maybe a plugin interface for exporters).

As such we can remove all of this it seems and turn on tracing based on `ExporterEndpoint` being set or not.